package.json

```diff
@@ -6,12 +6,19 @@
   "main": "src/index.js",
   "scripts": {
     "test": "npm run lint && node spec/index.js",
+    "test-node-modern": "npm run lint && node spec/nodeModernIndex.js",
     "build": "webpack",
-    "lint": "eslint src/* spec/*"
+    "lint": "eslint src/* spec/*",
+    "bench": "node bench/index.js",
+    "rebuild": "node-gyp rebuild",
+    "test-native": "node spec/native.js",
+    "debug-native": "lldb -- \"$(which node)\" spec/native.js"
   },
   "author": "David Schnurr",
   "license": "MIT",
   "devDependencies": {
+    "bindings": "^1.3.0",
+    "d3": "^4.2.2",
```

Comment:
Is d3 actually used anyhwere?

Reply:
No, missed that out when merging from upstream